src/libraries/System.Utf8String.Experimental/tests/System/ReflectionTests.netfx.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Tests
+{
+    public partial class ReflectionTests
+    {
+        [Fact]
+        public static void FormatterServices_GetUninitializedObject_DoesntThrow()
```

Comment:
Tests like this aren't strictly necessary. It's ok for the behavior to be undefined on downlevel platforms.

Reply:
Ok, I'll remove this test.